### ISSUE TITLE
Add About page

### ISSUE THREAD

**Issue by grote** [stevesoltys/seedvault#49]
In the main settings UI, I will add an About link to the three-dot overflow menu. When clicked, a DialogFragment opens showing information such as:

```
SeedVault Backup

A backup application using Android's internal backup API.

License: Apache2
Written by: Steve Soltys and Torsten Grote
Sponsored by: Calyx Institute for use in CalyxOS
Source Code: https://github.com/stevesoltys/seedvault
```

Let me know if there should be something else on that screen.

**Comment by stevesoltys** [comment]
Looks good to me!

**Comment by grote** [comment]
![device-2019-12-06-092844](https://user-images.githubusercontent.com/244947/70323155-d88bc680-180a-11ea-801f-7f4f4ff4ae65.png)

**Comment by grote** [comment]
Closed by #52
